jenkins/jmx-http-response-code-bad-request.yml

```diff
@@ -0,0 +1,12 @@
+type: jmx
+
+observation:
+  - name: jenkins-http-codes-bad-request
+    metricNamespace: jenkins
+    objectName: io.jenkins:type=Http,class=ResponseCodes,name=BadRequest
+
+    metric:
+      - name: http.response.code.bad.request.count
```

Comment:
I think you could name it just `http.response.bad-request.count`. And similar for others, e.g. `http.response.created.count`, `http.response.forbidden.count`, `http.response.no-content.count`... Maybe even drop `count`, since it is implied.